content/docs/overview/some_terms.md

```diff
@@ -22,9 +22,9 @@ If you right away want to deep-dive into the topic, you can have a look at the [
 
 ## What’s a Component Version?
 
-- Has an identity (name + version)
+- Has a globally unique `Component Identity` (identity attributes `name` + `version`)
```

Comment:
IMO, it might be hard to understand our usage of `component` and `component version`, especially considering we introduce yet another term (component identity) while explaining component version.

Also, comparing the definitions for `Component` and `Component Versions`, there is a mismatch in the presented aspect.

I suggest we should rather restructure this into a (short) text, highlighting terms like component, etc.

Like so:

```
## OCM's Notion of a Component

The term `component` is already overloaded, often by very specific and opinionated definitions considering granularity or other technical attributes. OCM deliberately focusses on the semantics and/or intended purpose of components, and does not take any assumptions about granularity or other technical attributes of *Components*.

In OCM, *Components* group a set of *Component Versions* ...
```

Reply:
@ccwienk, the overall intention was to have every term related to component(...) used in the later docs and especially the getting started and tutorials mentioned, without having to look in the spec. The content of this and the following terms was mainly taken (sometime even quoted) from your presentation in the video https://video.sap.com/media/t/1_v7rq3sma (minutes 12:30 - 20:00). Would you then like to continue your short text with similar textual explanation s of all terms below, instead of explaing them in bullet points or did you mean just to change the part for the `component`?